try:
    x = 3 / 0
    print(x)
except:
    print("An exception occurred.")
